### PR TITLE
feat(DMS): add datasource rocketmq flavors and update region property

### PR DESCRIPTION
**Pull request by 346010762**
**What this PR does / why we need it**:
   add datasource rocketmq flavors and update region property
**Special notes for your reviewer**:

**Release note**:

```release-note
add datasource rocketmq flavors and update region property
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/accepance/dms/ TESTARGS='-run TestAccDatasourceRocketmqFlavors_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDatasourceRocketmqFlavors_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceRocketmqFlavors_basic
=== PAUSE TestAccDatasourceRocketmqFlavors_basic
=== CONT  TestAccDatasourceRocketmqFlavors_basic
--- PASS: TestAccDatasourceRocketmqFlavors_basic (18.38s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       18.425s

make testacc TEST=./huaweicloud/services/acceptance/dms/ TESTARGS='-run TestAccKafkaFlavorsDataSource_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccKafkaFlavorsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccKafkaFlavorsDataSource_basic
=== PAUSE TestAccKafkaFlavorsDataSource_basic
=== CONT  TestAccKafkaFlavorsDataSource_basic
--- PASS: TestAccKafkaFlavorsDataSource_basic (17.83s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       17.868s

make testacc TEST=./huaweicloud/services/acceptance/dms/ TESTARGS='-run TestAccRabbitMQFlavorsDataSource_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccRabbitMQFlavorsDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccRabbitMQFlavorsDataSource_basic
=== PAUSE TestAccRabbitMQFlavorsDataSource_basic
=== CONT  TestAccRabbitMQFlavorsDataSource_basic
--- PASS: TestAccRabbitMQFlavorsDataSource_basic (18.65s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       18.694s
```

### CODE REVIEW

File: docs/data-sources/dms_rocketmq_flavors.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+* `support_features` - Indicates the list of features supported by the current specification.
+  The [support_features](#DMS_rocketmq_flavor_support_features) structure is documented below.
+
+* `type` - Indicates the type of the flavor. The value can be: **single** and **cluster**.
+
+* `properties` - Indicates the list of the properties of the current specification.
+  The [properties](#DMS_rocketmq_flavor_properties) structure is documented below.
+
+* `vm_specification` - Indicates the underlying resource type, e.g. **c6.large.2**
```
Comment (houpeng80) [inline docs/data-sources/dms_rocketmq_flavors.md]:
```suggestion
* `vm_specification` - Indicates the underlying VM specification, e.g. **c6.large.2**
```

Comment (346010762) [reply docs/data-sources/dms_rocketmq_flavors.md]: done

**Pull request merged**